helm/kubernetes-cluster-prep/values.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "properties": {
+        "conjur": {
+            "properties": {
+                "account": {
+                    "type": "string"
+                },
+                "applianceUrl": {
+                    "type": "string",
+                    "regex": "(^https:\/\/(.*))|(.*)"
+                },
+                "certificateFilePath": {
+                    "type": "string",
+                    "minLength": 1,
+                    "regex": "(^files\/(.*))|(^tests\/(.*))"
+                },
+                "certificateBase64": {
+                    "type": "string",
+                    "minLength": 1,
+                    "regex": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$"
+                }
+            }
+        },
+        "authnk8s": {
+            "properties": {
+                "authenticatorId": {
+                    "type": "string"
+                },
+                "clusterRole": {
+                    "properties": {
+                        "create": {
+                            "type": "boolean"
+                        },
+                        "name": {
```

Comment:
A max length check of 253 would be good here. I don't think the default K8s schema checks do this length check.